src/components/com_tjcertificate/site/views/certificate/view.html.php

```diff
@@ -41,27 +43,34 @@ class TjCertificateViewCertificate extends JViewLegacy
 	 */
 	public function display($tpl = null)
 	{
-		$input = Factory::getApplication()->input;
+		$params = ComponentHelper::getParams('com_tjcertificate');
+		$input  = Factory::getApplication()->input;
 
 		$this->uniqueCertificateId = $input->get('certificate', '', 'STRING');
-		$this->showSearchBox       = $input->getInt('show_search', 1);
+		$this->showSearchBox       = $input->getInt('show_search', $params->get('show_search_box'));
 		$this->tmpl                = $input->get('tmpl', '', 'STRING');
 
 		if (!empty($this->uniqueCertificateId))
 		{
 			$certificate = TJCERT::Certificate();
```

Comment:
can we not move this outside
`$certificate = TJCERT::Certificate();
` and use in both ifs insted of recreating instance?